### PR TITLE
Attempt at fixing compile problem

### PR DESCRIPTION
**Pull request by JoshuaSBrown**
Attempt to fix problem:

https://groups.google.com/forum/#!msg/votca/LGjDJLwuFgI/EnM4zFvJBQAJ

### CODE REVIEW

**Comment by JoshuaSBrown** [comment]
It looks like the pull request is building with master instead of stable...

File: include/votca/tools/graph.h [hunk 1]
```diff
@@ -124,8 +124,8 @@ class Graph : public EdgeContainer {
  *
  * sort(vec_pr_gn.begin(),vec_pr_gn.end(),cmpVertNodePair);
  */
-bool cmpVertNodePair(std::pair<int, GraphNode> gn1_pr,
-                     std::pair<int, GraphNode> gn2_pr);
-}
-}
+bool cmpVertNodePair(const std::pair<int, GraphNode> gn1_pr,
+                     const std::pair<int, GraphNode> gn2_pr);
```
Comment (JensWehner) [inline include/votca/tools/graph.h]: should be references not copies

File: src/libtools/graph.cc [hunk 2]
```diff
@@ -83,18 +83,18 @@ vector<int> Graph::getVerticesMissingNodes(void) {
...
+  for (auto neigh_vert : neigh_vertices) {
+    auto node_pr = pair<int, GraphNode>(neigh_vert, nodes_[neigh_vert]);
     neigh_vertices_pr.push_back(node_pr);
   }
   return neigh_vertices_pr;
 }
 
-void Graph::setNode(int vert, GraphNode gn){
-  if(nodes_.count(vert)){
+void Graph::setNode(int vert, GraphNode gn) {
```
Comment (JensWehner) [inline src/libtools/graph.cc]: should this be a copy, or if graph is not a pdo type this should be a const reference

File: src/libtools/graph.cc [hunk 3]
```diff
@@ -140,9 +140,10 @@ ostream& operator<<(ostream& os, const Graph g) {
   return os;
 }
 
-bool cmpVertNodePair(pair<int, GraphNode> gn1_pr, pair<int, GraphNode> gn2_pr) {
+bool cmpVertNodePair(const pair<int, GraphNode> gn1_pr,
```
Comment (JensWehner) [inline src/libtools/graph.cc]: this should be a const reference

File: src/libtools/graph.cc (hunk 2 above)
Comment (JoshuaSBrown) [reply src/libtools/graph.cc]: what is 'pdo' ?

Comment (JoshuaSBrown) [reply src/libtools/graph.cc]: I'm not fixing this in stable, because it might start me down a rabbit hole and it is not what is causing the error.

**Comment by JoshuaSBrown** [comment]
gitlab still building against master...

**Comment by junghans** [comment]
The branch name needs to be `for/stable/...`, but don't worry I just push the commit on another branch, so that should trigger the build without the need for a new PR.

**Comment by JoshuaSBrown** [comment]
oh right, I forgot. My bad.

**Pull request merged**